### DISCUSSION TITLE
Bug in the "Plugin Registration" Plugin

### DISCUSSION THREAD

**Discussion by Dashulol** [MscrmTools/XrmToolBox#1138] [General]
Hi,

this is obviously the wrong place but unfortunately there doesn't seem to be a better way to get in contact. The plugin has no useful project page and no contact information for the devs so I'm not sure if it's even still in development. Though in the current form it probably shouldn't be used anymore.

The Plugin Registration Tool integrated into XrmToolbox does not use the latest version of the official Plugin Registration Tool and is therefore unable to handle MultiSelectOptionSet fialters on plugin steps. That support was added in version 9.0.2.5. Now if you update plugin steps using the XrmToolbox Plugin, those filters just get deleted silently.

**Comment by MscrmTools** [comment]
The repo is here : https://github.com/Innofactor/PluginRegistration

You can add an issue
